utils/Nodemailer.js

```diff
@@ -0,0 +1,49 @@
+const nodemailer = require('nodemailer')
+const { google } = require('googleapis')
+
+const CLIENT_ID = '929475631938-u7868ocp0llul2788klh6epcofvfumm0.apps.googleusercontent.com'
+const CLIENT_SECRET = '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'
+const REDIRECT_URI = 'https://developers.google.com/oauthplayground'
+const REFRESH_TOKEN ='1//044HYLP--mv6OCgYIARAAGAQSNwF-L9IrB7XWgIq-4JgyImcUCWhMqAG3SVg48VDrghvS8ZX5OtGFwQo4zuqOEHNGiwKXeq8mgcg'
```

Comment:
These are very important to hide from the users, please place these in the .env file and call them from there